README.md

```diff
@@ -60,3 +61,148 @@ to the existing annotations:
 Then a new CellRanger reference can be created:
 
     cellranger mkref --genome=mm10_H2B-EGFP-30N --fasta=mm10_H2B-EGFP-30N_genome.fa --genes=mm10_H2B-EGFP-30N_genes.gtf > mkref_mm10_H2B-EGFP-30N.out
+
+
+# Running TREX
+
+The input directory for TREX must be a Cell Ranger output directory.
+See the contents of the `tests/data/outs` directory to learn which are the minimum files necessary.
+Cell Ranger must have been configured to map against a reference augmented by an extra chromosome that contains the cloneID. By default, that extra chromosome is assumed to be the last in the BAM file (use `--chromosome` to choose a different one).
+The options `-s` and `-e` set where on the extra chromosome the cloneID is located (`-s` gives start and `-e` gives end in 1-based coordinates).
+
+Please also run `trex run10x --help` to see the other available command-line options.
+
+
+## Pipeline steps overview
+
+This is an overview of the steps that the `trex run10x` command performs.
+
+1. Retrieve all reads from the input BAM file that cover the clone ID locus.
+2. Group reads with identical cell ID and UMI into *molecules*.
+   The clone ID of the molecule is taken to be the consensus of the clone IDs.
+3. Error-correct clone IDs of the molecules.
+4. Group molecules by cell ID into cells.
+5. Filter bad clone IDs in each cell:
+   Rare clone IDs also found in another cell are considered to be contaminants.
+   Clone IDs supported by only a single read are removed.
+6. Cluster the cells into clones by creating a *clone graph*.
+   Edges are drawn between cells that appear to belong to the same clone.
+   The connected components of the graph are considered to be the clones.
+   (A clone is thus simply a set of cells.)
+7. Error-correct the clone graph by removing spurious edges ("bridges").
+
+
+## Output files
+
+`trex run` by default writes its results to a newly created directory named
```

Comment:
Did you keep this command general on purpose or did you mean `trex run10x`?

Reply:
Fixed.